db/migrate/20200108062848_add_name_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddDisplayNameToUsers < ActiveRecord::Migration[6.0]
```

Comment:
This is killing the heroku build.  I'll fix it

Reply:
<3